Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -7,121 +7,604 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import Dcrlibwallet
+import Signals
 
-class OverviewViewController: UIViewController {
-    @IBOutlet weak var syncProgressViewContainer: UIView!
+class OverviewViewController: UIViewController{
+    // Heading & balance
+    @IBOutlet weak var logoImage: UIImageView!
+    @IBOutlet weak var overViewLabel: UILabel! {
+        didSet { self.overViewLabel.text = LocalizedStrings.overview }
+    }
+    @IBOutlet weak var balanceLabel: UILabel! {
+        didSet { self.balanceLabel.font = UIFont(name: "Source Sans Pro", size: 40) }
+    }
+    @IBOutlet weak var currentBalanceLabel: UILabel! {
+        didSet {
+            self.currentBalanceLabel.text = LocalizedStrings.currentTotalBalance
+            self.currentBalanceLabel.font = UIFont(name: "Source Sans Pro", size: 14)
+        }
+    }
     
-    @IBOutlet weak var overviewPageContentView: UIView!
-    @IBOutlet weak var fetchingBalanceIndicator: UIImageView!
-    @IBOutlet weak var totalBalanceLabel: UILabel!
-    @IBOutlet weak var recentActivityTableView: UITableView!
+    // Backup phrase section (Top view)
+    @IBOutlet weak var backupWarning: UIView!
```

Comment:
I guess this refers to the parent container for all the views in that section. If so, `seedBackupSectionView` will be a better variable name.

Reply:
resolved in 9991341